pom.xml

```diff
@@ -46,27 +46,12 @@
       <dependency>
         <groupId>org.slf4j</groupId>
         <artifactId>slf4j-api</artifactId>
-        <version>2.0.6</version>
+        <version>${slf4j.version}</version>
```

Comment:
Consuming the new property in the original consumer. It will also be consumed later on in a new consumer (`slf4j-jdk14`).